module-options.nix

```diff
@@ -153,38 +153,43 @@ in
         type = types.functionTo types.package;
         defaultText = lib.literalMD "Default check implementation";
         default = self:
-          pkgs.runCommandLocal "treefmt-check"
-            {
-              buildInputs = [ pkgs.git config.build.wrapper ];
-            }
-            ''
-              set -e
-              # `treefmt --fail-on-change` is broken for purs-tidy; So we must rely
-              # on git to detect changes. An unintended advantage of this approach
-              # is that when the check fails, it will print a helpful diff at the end.
-              PRJ=$TMP/project
-              cp -r ${self} $PRJ
-              chmod -R a+w $PRJ
-              cd $PRJ
-              export HOME=$TMPDIR
-              cat > $HOME/.gitconfig <<EOF
-              [user]
-                name = Nix
-                email = nix@localhost
-              [init]
-                defaultBranch = main
-              EOF
-              git init
-              git add .
-              git commit -m init --quiet
-              export LANG=C.UTF-8
-              export LC_ALL=C.UTF-8
-              treefmt --version
-              treefmt --no-cache
-              git status
-              git --no-pager diff --exit-code
-              touch $out
-            '';
+          let
+            check = pkgs.runCommandLocal "treefmt-check"
+              {
+                buildInputs = [ pkgs.git config.build.wrapper ];
```

Comment:
Couldn't the meta.description be added in here directly?

Reply:
Oh right. I got so used to various builders not accepting meta (eg: https://github.com/NixOS/nixpkgs/pull/251369) that I didn't think `runCommand` did. It's done now.